### ISSUE TITLE
Some failure transitions missing?

### ISSUE THREAD

**Issue by mganss** [alexandrnikitin/AhoCorasick.Net#1]
Consider this input:

``` C#
var sut = new AhoCorasickTree(new[] { "abcd", "bc" });
var x = sut.Contains("abc"); // => false
```

This should yield true because "abc" contains "bc". The failure transition from "c" of the "abcd" subtree to the "c" of "bc" seems to be missing.

**Comment by alexandrnikitin** [comment]
No, it shouldn't. There's no keyword "abc" in the Aho-Corasick dictionary. You specified "abcd" and "bc". It searches those keywords in the specified string which is "abc" in your case. Yes, `Contains()` could be misleading here. Other ideas?

**Comment by alexandrnikitin** [comment]
Take a look at [the example](https://github.com/alexandrnikitin/AhoCorasick.Net#example-usage)

**Comment by alexandrnikitin** [comment]
Ops, you're right! Sorry. I'll take a look.

**Comment by alexandrnikitin** [comment]
Released 0.5.0
